.circleci/config.yml

```diff
@@ -17,7 +17,7 @@ macos_config: &macos_defaults
     xcode: "10.1.0"
   working_directory: /Users/distiller/project/src/github.com/cosmos/cosmos-sdk
   environment:
-    GO_VERSION: "1.12.1"
+    GO_VERSION: "1.12.4"
```

Comment:
I think we should move this to an alias on top of the file, so we reduce the risk of forgetting one of those three points over time

Reply:
Would you help doing it in an ensuing PR please?